docs/resources/Guild.md

```diff
@@ -36,11 +39,46 @@ Guilds in Discord represent a collection of users and channels into an isolated
 
 ** \* These fields are only sent within the [GUILD_CREATE](#DOCS_GATEWAY/guild-create) event **
 
+###### Default Message Notification Level
+
+| Key | Value |
+| --- | ----- |
+| ALL_MESSAGES | 0 |
+| ONLY_MENTIONS | 1 |
+| NO_MESSAGES| 2 |
+| NULL | 3 |
+
+###### Explicit Content Filter Level
+
+| Level | Integer
+| ---- | ------ |
+| DISABLED | 0 |
+| MEMBERS_WITHOUT_ROLES | 1 |
+| ALL_MEMBERS | 2 |
+
+###### MFA Level
+
+| Level | Integer |
+| ----- | ------- |
+| NONE | 0 |
+| ENABLED | 1 |
```

Comment:
s/ENABLED/ELEVATED/

Elevated = requires 2fa for actions that use the following permission: 
```
    KICK_MEMBERS,
    BAN_MEMBERS,
    ADMINISTRATOR,
    MANAGE_CHANNELS,
    MANAGE_GUILD,
    MANAGE_ROLES,
    MANAGE_MESSAGES,
    MANAGE_WEBHOOKS,
```